templates/import.php

```diff
@@ -0,0 +1,269 @@
+<?php
+/**
+ * Page for the actual import step.
+ */
+?>
+<div class="welcome-panel">
+	<div class="welcome-panel-content">
+		<h2><?php esc_html_e( 'Step 3: Importing', 'wordpress-importer' ) ?></h2>
+		<p id="import-status-message"><?php esc_html_e( 'Now importing.', 'wordpress-importer' ) ?></p>
+
+		<table class="import-status">
+			<thead>
+				<tr>
+					<th><?php esc_html_e( 'Import Summary', 'wordpress-importer' ) ?></th>
+					<th><?php esc_html_e( 'Completed', 'wordpress-importer' ) ?></th>
+					<th><?php esc_html_e( 'Progress', 'wordpress-importer' ) ?></th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>
+						<span class="dashicons dashicons-admin-post"></span>
+						<?php echo esc_html( sprintf(
+							_n( '%d post (including CPTs)', '%d posts (including CPTs)', $data->post_count, 'wordpress-importer' ),
```

Comment:
Should we abbreviate here? If it's user facing CPT is kind of a dev shorthand term